doc/markdown/Intructions.md

```diff
@@ -21,6 +21,46 @@ Response: Where is it located?
 
 ## Dataset management functions
 
+### Subset of dataset
+
+#### \- By row
+
+* select by rows from **1** to **10** from **energy**
+* Get a subset dataset of **energy**
+* Obtain a subset of **energy** by rows from **25** to **60**
+
+#### \- By column
+
+* Obtain a subset of **random0** by columns **col0**, **col1**, **col2** and **col5**
+* Get a subset of **energy** by columns
+* select by columns **index** from **stomp0**
```

Comment:
reformulate this one to: (remove by)

select columns **index** from **stomp0**